sectionsUtilityTool/README.txt

```diff
@@ -4,15 +4,23 @@
  
  a) sectionsTool$ mvn clean install
  b) copy to tomcat/webapp
- c) Add the property file on your linux box sectionsToolProps.properties. Then in JAVA_OPTS -DsectionsToolPropsPath=file:/Users/pushyami/sections/sectionsToolProps.properties 
- d) Run this as http://localhost:port/sectionsUtilityTool
+ c) Add 2 property files on linux box sectionsToolPropsSecure.properties and sectionsToolPropsLessSecure.properties Then in JAVA_OPTS add  2 -D's 
+		 1) -DsectionsToolPropsPathSecure=file:/<file-path>/sectionsToolPropsSecure.properties 
+		 2) -DsectionsToolPropsPathLessSecure=file:/<file-path>/sectionsToolPropsLessSecure.properties 
+ d) Run this as http://localhost:port/sectionsUtilityTool/?testUser=<uniquename>.
+     no need of testUser in Prod as we will enable the cosign for authentication and we will be getting remote user info from that.
  
-2) [sectionsToolProps.properties]
- Add the following 2 properties to this file. 
+2) [sectionsToolPropsSecure.properties]
+ Add the following 3 properties to this file. 
```

Comment:
For the security file we have often create a file like sectionsToolPropsSecure.properties.template to check into source control.  This means that an installer has an almost valid sample properties file that they just need to rename and fill in.  The source control system should be configured to not allow checking in a real properties file.  For git this can be done by adding a .gitignore file.